src/components/search/detailed/NameLink.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Highlighter } from "@cyverse-de/ui-lib";
+import { Link as MuiLink, makeStyles } from "@material-ui/core";
```

Comment:
Since this is the only `Link` in this file, I don't think it has to be renamed `MuiLink` now.
Feel free to leave it if you think there still might be any confusion with a `next/link`, though.